src/services/Store/store/account.slice.ts

```diff
@@ -1,3 +1,4 @@
+import { BigNumber as EthersBN } from '@ethersproject/bignumber';
```

Comment:
Any reason not to use `bigify` ?

Reply:
We use `EthersBN` for tx values at the moment.